### PR TITLE
[CU-86860dyr2] Adds reward bonus for user reports

### PR DESCRIPTION
**Pull request by manuelurenah**

### CODE REVIEW

**Comment by JustMaier** [comment]
Task linked: [CU-86860dyr2 Add Buzz Rewards](https://app.clickup.com/t/86860dyr2)

**Comment by JustMaier** [comment]
Task linked: [CU-86860dyr2 Add Buzz Rewards](https://app.clickup.com/t/86860dyr2)

File: src/server/rewards/active/reportAccepted.reward.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import { createBuzzEvent } from '../base.reward';
+
+const type = 'reportAccepted' as const;
+export const reportAcceptedReward = createBuzzEvent({
+  type,
+  description: 'For each report you make that is accepted',
+  awardAmount: 5,
+  cap: 25,
```
Comment (manuelurenah) [inline src/server/rewards/active/reportAccepted.reward.ts:8]: We need to confirm what would be amount and cap

**Review by lrojas94 — APPROVED** [review]

File: src/server/rewards/passive/reportAccepted.reward.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { createBuzzEvent } from '../base.reward';
+
+const type = 'reportAccepted' as const;
+export const reportAcceptedReward = createBuzzEvent({
+  type,
+  description: 'For each report you make that is accepted',
+  awardAmount: 50,
+  caps: [{ amount: 1000, interval: 'month', keyParts: ['toUserId'] }],
```
Comment (manuelurenah) [inline src/server/rewards/passive/reportAccepted.reward.ts:8]: @JustMaier There's no easy way to actually make this an active reward atm, cause all `onDemand` rewards are reseted daily. So, I moved it to be passive but I believe we need to make adjustments to actually apply it, which I'm not familiar with. Any guidance you can give will help 🙏

**Pull request merged**